docs/sources/reference/api/docker_remote_api_v1.17.md

```diff
@@ -124,6 +124,11 @@ Create a container
              "OpenStdin":false,
              "StdinOnce":false,
              "Env":null,
+             "Labels": [
+                     "Vendor=Acme",
+                     "License=GPL",
+                     "Version=1.0"
+             ],
```

Comment:
Is this a list for a specific reason, or just consistency with `Env`? Could it be this?

"Labels": {
  "Vendor": "Acme",
  "License": "GPL",
  "Version": "1.0"
},